.gitignore

```diff
@@ -10,3 +10,5 @@ venv/
 .Rhistory
 .Rproj.user/
 .venv/
+/.idea/
```

Comment:
Could we keep editor-specific ignore lines in global ignores?  Saves literally every project file with them 🙂